manifests/python.yml

```diff
@@ -1287,6 +1279,7 @@ tests/:
     Test_FFE_Exposure_Events: v4.0.0
     Test_FFE_Exposure_Events_Empty: v4.0.0
     Test_FFE_Exposure_Events_Errors: v4.0.0
+    Test_FFE_RC_Unavailable: missing_feature
```

Comment:
Disabling until the remote configuration client issue is resolved: https://github.com/DataDog/system-tests/actions/runs/19725812231/job/56517122931?pr=5763